spec/features/journeys/basic_registration_journey_spec.rb

```diff
@@ -30,6 +40,10 @@
       page.choose("Yes", visible: :all)
     end
 
+    unless js
+      expect_page_to_have(path: "/registration/get-an-identity", submit_form: true)
+    end
```

Comment:
Extra non-JS only step. Not a fan of a leading `unless` but when it's inline it might get lost over there on the right!